app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrDeleteSubscription.java

```diff
@@ -0,0 +1,66 @@
+package org.pesmypetcare.mypetcare.controllers;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.features.community.Group;
+import org.pesmypetcare.mypetcare.features.community.GroupNotExistingException;
+import org.pesmypetcare.mypetcare.features.community.NotSubscribedException;
+import org.pesmypetcare.mypetcare.features.community.OwnerCannotDeleteSubscriptionException;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubCommunityService;
+import org.pesmypetcare.usermanagerlib.datacontainers.DateTime;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Xavier Campos
+ */
+public class TestTrDeleteSubscription {
+    private User user;
+    private Group group;
+    private TrDeleteSubscription trDeleteSubscription;
+
+    @Before
+    public void setUp() {
+        user = new User("John Smith", "johnSmith@gmail.com", "1234");
+        group = new Group("Husky", "John Doe", DateTime.Builder.buildDateString("2020-04-15"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "2020-04-15" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5389854)